packages/core/src/tests/tooltip/table/index.html

```diff
@@ -0,0 +1,42 @@
+
+
+<!--
+SPDX-FileCopyrightText: 2024 Siemens AG
+
+SPDX-License-Identifier: MIT
+-->
+
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0"
+    />
+    <title>Stencil Component Starter</title>
+  </head>
+  <body style="padding: 12rem">
+    <div style="padding: 4rem; width: 300px" }}>
+      <table class="table">
+        <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">First</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <th scope="row">1</th>
+          <td class="td1">Table data</td>
+        </tr>
+        <div>
+          <ix-tooltip for=".td1">Simple selector</ix-tooltip>
```

Comment:
Do a resizing of the tooltip (width and height) and overwrite the background via css, a complete table is here not necessary. `<ELEMENT style="....">` do the same as the table selector `table > *`

```html
<button id="test">Test</button>
<ix-tooltip for="#test" style="background-color: red; width: 25rem; height: 25rem">My tooltip</ix-tooltip>
```